docs/UserGuide.md

```diff
@@ -128,24 +128,22 @@ Examples:
 * `editc 2 n/Betsy Crower t/` Edits the name of the 2nd contact to be `Betsy Crower` and clears all existing tags.
 
 
-### Locating contacts by name: `find`
+### Locating contacts by name: `findc`
```

Comment:
The `findc` command is no longer limited to locating by name only, so do remember to reflect that here as well